builder/config/load.go

```diff
@@ -1,30 +1,31 @@
 package config
 
 import (
-	"fmt"
-
 	"github.com/spf13/viper"
 
 	"github.com/uor-framework/client/builder/api/v1alpha1"
 )
 
-func ReadConfig(configName string) (c v1alpha1.DataSetConfiguration, err error) {
+func ReadConfig(configName string) (v1alpha1.DataSetConfiguration, error) {
+	var configuration v1alpha1.DataSetConfiguration
 
 	viper.SetConfigName(configName)
 	viper.AddConfigPath(".")
 	viper.SetConfigType("yaml")
 
-	var configuration v1alpha1.DataSetConfiguration
-
-	err = viper.ReadInConfig()
+	err := viper.ReadInConfig()
 	if _, ok := err.(viper.ConfigFileNotFoundError); ok {
```

Comment:
Seems like we don't need this statement. We return the error either way.

Reply:
Yeah, that was dumb